core/src/main/java/org/elasticsearch/common/xcontent/XContent.java

```diff
@@ -49,6 +49,11 @@
     XContentGenerator createGenerator(OutputStream os, String[] filters) throws IOException;
 
     /**
+     * Creates a new generator using the provided output stream and some
+     * filters.
+     */
+    XContentGenerator createGenerator(OutputStream os, String[] filters, boolean inclusiveFilters) throws IOException;
```

Comment:
I think that `inclusiveFilters` could be renamed to `inclusive` so that it's consistent with other layers.

Can we also add javadoc here?